### PR TITLE
added bio for jcollinsdg

### PR DESCRIPTION
**Pull request by jcollinsdg**
@githubstudent what do you know?

### CODE REVIEW

File: bios/jcollinsdg-bio.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+## Simple Bio Data
+
+- item 1
```
Comment (jcollinsdg) [inline bios/jcollinsdg-bio.md:3]: What is this?

Comment (jcollinsdg) [inline bios/jcollinsdg-bio.md:3]: This is an interesting item.  The number one.

Comment (jcollinsdg) [inline bios/jcollinsdg-bio.md:3]: Something else

**Comment by bryanc75** [comment]
:+1:

**Pull request merged**